### PR TITLE
space docs

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

**Review by whitewum — APPROVED** [review]

File: docs_2.0/manual_EN/4.ngql-guide/9.space-statements.md/3.show-spaces.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+```ngql
+nebula> SHOW SPACES;
++--------+
+| Name   |
++--------+
+| "nba"  |
++--------+
+```
+
+To create graph spaces, see the [Create Space document](1.create-space.md).
```
Comment (randomJoe211) [inline docs_2.0/manual_EN/4.ngql-guide/9.space-statements.md/3.show-spaces.md:20]: "the" is not needed. Please delete it.

**Review by randomJoe211 — APPROVED** [review]
👍

**Review by whitewum — APPROVED** [review]

File: docs_2.0/manual_EN/4.ngql-guide/9.space-statements.md/4.describe-space.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# DESCRIBE SPACE
+
+```ngql
+DESCRIBE SPACE <graph_space_name>
```
Comment (whitewum) [inline docs_2.0/manual_EN/4.ngql-guide/9.space-statements.md/4.describe-space.md:4]: is `DESC SPACE nba` OK?

**Review by whitewum — APPROVED** [review]

File: docs_2.0/manual_EN/4.ngql-guide/9.space-statements.md/4.describe-space.md (hunk 2 above)
Comment (Amber1990Zhang) [reply docs_2.0/manual_EN/4.ngql-guide/9.space-statements.md/4.describe-space.md:4]: ok

File: docs_2.0/manual_EN/4.ngql-guide/9.space-statements.md/5.drop-space.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# DROP SPACE
+
+```ngql
+DROP SPACE [IF EXISTS] <graph_space_name>
+```
+
+The `DROP SPACE` statement deletes everything in the related graph space.
```
Comment (randomJoe211) [inline docs_2.0/manual_EN/4.ngql-guide/9.space-statements.md/5.drop-space.md:7]: Delete "related". It sounds like all the related graph spaces will be dropped.

**Review by randomJoe211 — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Pull request merged**